docker/build_docker.sh

```diff
@@ -242,6 +248,7 @@ if [[ ! $(which pyenv) ]] && [[ -n "${CIRCLECI}" ]]; then
     export PATH="$HOME/.pyenv/bin:$PATH"
     eval "$(pyenv init -)"
     eval "$(pyenv virtualenv-init -)"
+    pyenv shell system $(pyenv versions --bare | grep 3.7)
```

Comment:
why 3.7 and not 3.8?

Reply:
I want to make sure the script is compatible with 3.7